app/main.py

```diff
@@ -14,0 +1,13 @@
+from models import Actor
+from managers import ActorManager
+
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+    Actor.objects.create(first_name="Emma", last_name="Watson")
+    Actor.objects.create(first_name="Daniel", last_name="Radclife")
+    print(Actor.objects.all())
+    Actor.objects.update(2, "Daniel", "Radcliffe")
```

Comment:
The `update` method should be called with the correct order of parameters: `id_to_update`, `first_name`, `last_name`. Ensure that the ID is the first parameter.